lib/stat_tracker.rb

```diff
@@ -56,6 +55,13 @@ def average_goals_per_game
     (goals.to_f / total_games).round(2)
   end
 
+  def average_goals_by_season
```

Comment:
Ooh, group_by, nice. That'll probably prove useful for me in count_of_games_by_season